### ISSUE TITLE
Can't configure or select VNDB as a metadata source.

### ISSUE THREAD

**Issue by Ryah** [Mithnar/PlayniteVndb#5]
Extension enabled and installed as instructed, but playnite isn't detecting it as a metadata source.

IGDB is the only source under Metadata Sources in the settings.

I'm using the latest version of Playnite (6.4).

Screenshot: https://owo.whats-th.is/3dAhovd.png

**Comment by Mithnar** [comment]
Are you using the latest release of the VNDB plugin?

Could you please (optionally, to have a clean log) delete your playnite.log file, then restart playnite and attach the log file to the issue? (playnite.log is found in your playnite directory)

**Comment by Ryah** [comment]
I was able to fix it. Turns out Playnite is installed in both %Appdata%/Roaming AND %Appdata%/Local. My executable was loading from the Roaming folder, but the extension needed to be installed in the Local folder, even though both of them had an extensions folder.

Also, the extensions you listed in your comment for me are in the Local folder, but the ExtensionsData is in the Roaming folder. The log also showed that it was trying to load from the Roaming folder.

https://owo.whats-th.is/9c4pkRk.png

https://owo.whats-th.is/69Sonpy.png

I also had a hard time finding the log until I ran across the Roaming folder, so that helped a lot too lol.
